webapp/components/admin_console/session_settings.jsx

```diff
@@ -90,14 +90,14 @@ export default class SessionSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.ssoSessionDays'
-                            defaultMessage='Session Length for SSO in Days:'
+                            defaultMessage='Session length for GitLab SSO authentication (days):'
                         />
                     }
                     placeholder={Utils.localizeMessage('admin.service.sessionDaysEx', 'Ex "30"')}
                     helpText={
                         <FormattedMessage
                             id='admin.service.ssoSessionDaysDesc'
-                            defaultMessage='The SSO session will expire after the number of days specified and will require a user to login again.'
+                            defaultMessage='GitLab SSO sessions will expire after the number of days specified and will require users to login again.'
```

Comment:
Propose "GitLab single-sign-on sessions will expire after the number of days specified and will require users to sign-in again.